OMPython/docker/Dockerfile.ompython.linux64.focal

```diff
@@ -0,0 +1,35 @@
+# Build with
+# $ docker build --tag docker.openmodelica.org/ompython:focal -f Dockerfile.ompython.linux64.focal .
+FROM ubuntu:focal
+
+# Get rid of the warning: "debconf: unable to initialize frontend: Dialog"
+# https://github.com/moby/moby/issues/27988
+RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
+
+# Install git, wget, Python 3 + pandas
+RUN apt-get update \
+    && apt-get install git python3-pip wget -qy \
+    && pip3 install pandas \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+# Install OMC and OMPython
+RUN for deb in deb deb-src; do echo "$deb http://build.openmodelica.org/apt focal stable"; done | tee /etc/apt/sources.list.d/openmodelica.list \
+    && wget -q http://build.openmodelica.org/apt/openmodelica.asc -O- | apt-key add - \
+    # To verify that your key is installed correctly
+    && apt-key fingerprint \
+    # Gives output:
+    # pub   2048R/64970947 2010-06-22
+    #      Key fingerprint = D229 AF1C E5AE D74E 5F59  DF30 3A59 B536 6497 0947
+    # uid                  OpenModelica Build System
+    && apt-get update \
+    && apt-get install omc -qy \
+    && python3 -m pip install -U https://github.com/OpenModelica/OMPython/archive/master.zip \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+# Add non-root user
+RUN groupadd -g 1000 testUser \
+    && useradd -r -u 1000 -g testUser testUser
+USER testUser
+ENV USER=testUser
```

Comment:
Again, don't create users in the image. Use arguments to docker to setup the user instead.